CHANGELOG.md

```diff
@@ -1,5 +1,144 @@
+* [`trio-unneeded-sleep`](https://docs.astral.sh/ruff/rules/trio-unneeded-sleep/): `TRIO110` to `ASYNC110`
+* [`trio-zero-sleep-call`](https://docs.astral.sh/ruff/rules/trio-zero-sleep-call/): `TRIO115` to `ASYNC115`
+* [`repeated-isinstance-calls`](https://docs.astral.sh/ruff/rules/repeated-isinstance-calls/): `PLR1701` to `SIM101`
+
+### Stabilization
+
+The following rules have been stabilized and are no longer in preview:
+
+- [`mutable-fromkeys-value`](https://docs.astral.sh/ruff/rules/mutable-fromkeys-value/)
+- [`default-factory-kwarg`](https://docs.astral.sh/ruff/rules/default-factory-kwarg/)
+- [`django-extra`](https://docs.astral.sh/ruff/rules/django-extra/)
+- [`manual-dict-comprehension`](https://docs.astral.sh/ruff/rules/manual-dict-comprehension/)
+- [`print-empty-string`](https://docs.astral.sh/ruff/rules/print-empty-string/)
+- [`readlines-in-for`](https://docs.astral.sh/ruff/rules/readlines-in-for/)
+- [`if-expr-min-max`](https://docs.astral.sh/ruff/rules/if-expr-min-max/)
+- [`bit-count`](https://docs.astral.sh/ruff/rules/bit-count/)
+- [`redundant-log-base`](https://docs.astral.sh/ruff/rules/redundant-log-base/)
+- [`regex-flag-alias`](https://docs.astral.sh/ruff/rules/regex-flag-alias/)
+- [`isinstance-type-none`](https://docs.astral.sh/ruff/rules/isinstance-type-none/)
+- [`type-none-comparison`](https://docs.astral.sh/ruff/rules/type-none-comparison/)
+- [`implicit-cwd`](https://docs.astral.sh/ruff/rules/implicit-cwd/)
+- [`hashlib-digest-hex`](https://docs.astral.sh/ruff/rules/hashlib-digest-hex/)
+- [`list-reverse-copy`](https://docs.astral.sh/ruff/rules/list-reverse-copy/)
+- [`bad-open-mode`](https://docs.astral.sh/ruff/rules/bad-open-mode/)
+- [`empty-comment`](https://docs.astral.sh/ruff/rules/empty-comment/)
+- [`global-at-module-level`](https://docs.astral.sh/ruff/rules/global-at-module-level/)
+- [`misplaced-bare-raise`](https://docs.astral.sh/ruff/rules/misplaced-bare-raise`/)
+- [`non-ascii-import-name`](https://docs.astral.sh/ruff/rules/non-ascii-import-name/)
+- [`non-ascii-name`](https://docs.astral.sh/ruff/rules/non-ascii-name/)
+- [`nonlocal-and-global`](https://docs.astral.sh/ruff/rules/nonlocal-and-global/)
+- [`potential-index-error`](https://docs.astral.sh/ruff/rules/potential-index-error/)
+- [`redeclared-assigned-name`](https://docs.astral.sh/ruff/rules/redeclared-assigned-name/)
+- [`redefined-argument-from-locals`](https://docs.astral.sh/ruff/redefined-argument-from-locals/)
+- [`repeated-keyword-argument`](https://docs.astral.sh/ruff/rules/repeated-keyword-argument/)
+- [`super-without-brackets`](https://docs.astral.sh/ruff/rules/super-without-brackets/)
+- [`unnecessary-list-index-lookup`](https://docs.astral.sh/ruff/rules/unnecessary-list-index-lookup/)
+- [`useless-exception-statement`](https://docs.astral.sh/ruff/rules/useless-exception-statement/)
+- [`useless-with-lock`](https://docs.astral.sh/ruff/rules/useless-with-lock/)
+
+The following behaviors have been stabilized:
+
+TODO
```

Comment:
````suggestion
- `invalid-literal-comparison` (`F632`) now
  [also warns for identity checks against list, set or dictionary literals](https://github.com/astral-sh/ruff/pull/8607).
  For example, antipatterns such as `if x is {}` or `if y is ["foo"]` will now be identified by the check.

- `needless-bool` (`SIM103`)
  [now detects `if` expressions with implicit `else` branches](https://github.com/astral-sh/ruff/pull/10414).
  For example, the following snippet would not have been previously flagged by the rule, but will be in Ruff v0.5.0+:

  ```python
  def is_greater_than_ten(number: int) -> bool:
      if number > 10:
          return True
      return False
  ```

- `module-import-not-at-top-of-file` (`E402`)
  [now allows `os.environ` modifications between import statements](https://github.com/astral-sh/ruff/pull/10066).

- `type-comparison` (`E721`)
  [now allows idioms such as `type(x) is int`](https://github.com/astral-sh/ruff/pull/7905).

- `yoda-condition` (`SIM300`)
  [now flags a wider range of expressions](`https://github.com/astral-sh/ruff/pull/9164).
  For example, expressions such as `[] == make_me_a_list()` would previously not have triggered `yoda-condition`,
  but will in Ruff v0.5.0+.
````

Reply:
I prefer the more brief version

Reply:
Agreed, for a changelog what you have is better!